migrate/20250226_project_id_location_indexes_drop.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  no_transaction
+
+  up do
+    %i[postgres_resource firewall private_subnet vm minio_cluster kubernetes_cluster].reverse_each do |table|
+      drop_index table, [:project_id, :location, :name], name: :"#{table}_project_id_location_name_uidx", unique: true, concurrently: true
```

Comment:
note this also drops a constraint, right? that is, the uniqueness constraint? so it could break some concurrent algorithms, if there are any.

Reply:
I removed the migration file. The constraint is removed when we remove the location column anyway.